src/amo/components/SearchFilters/index.js

```diff
@@ -30,6 +31,7 @@ import Select from 'ui/components/Select';
 import './styles.scss';
 
 const NO_FILTER = '';
+const sortSelectName = 'sort';
```

Comment:
Could you use `paramsToFilter.sort` from `src/core/searchUtils.js` instead?

Reply:
Good idea. I think I'll keep `sortSelectName` but set it to `paramsToFilter.sort`. I'm not sure if you meant to get rid of it entirely and just use `paramsToFilter.sort` in the code, but I think having the name makes it clearer that there's a relationship between the two uses.

Reply:
It's fine to keep `sortSelectName` :)